x-pack/plugin/stack/src/javaRestTest/java/org/elasticsearch/xpack/stack/EcsDynamicTemplatesIT.java

```diff
@@ -76,7 +76,7 @@ private static void prepareEcsDynamicTemplates() throws IOException {
         String rawEcsComponentTemplate = TemplateUtils.loadTemplate(
             "/" + ECS_DYNAMIC_TEMPLATES_FILE,
             Integer.toString(1),
-            StackTemplateRegistry.TEMPLATE_VERSION_VARIABLE,
+            LegacyStackTemplateRegistry.TEMPLATE_VERSION_VARIABLE,
```

Comment:
Why is this test using the legacy one? Why not use the newly renamed files?

Reply:
That's an oversight.